deploy/clowdapp.yaml

```diff
@@ -135,7 +135,7 @@ parameters:
 - name: INGRESS_STAGEBUCKET
   value: insights-upload-perma
 - name: INGRESS_VALIDTOPICS
-  value: advisor,compliance,hccm,qpc,rhv,tower,leapp-reporting,topological-inventory,buckit,xavier,mkt,catalog,playbook,playbook-sat,resource-optimization,malware-detection,pinakes,assisted-installer
+  value: advisor,compliance,hccm,qpc,rhv,tower,leapp-reporting,topological-inventory,buckit,xavier,mkt,catalog,playbook,playbook-sat,resource-optimization,malware-detection,pinakes,assisted-installer,runtimes-java-general,runtimes-java-eap
```

Comment:
@kittylyst Can you provide a little more context on what each topic would be used for?  Is the intent that different processes would be responsible for handling different types of data being sent to ingress?

Reply:
@gmcculloug At this point, we don't have enough detailed design to be able to answer that question. This is for staging only right now - not expected to go into production. is it easier to just have a single `runtimes-java-general` topic for now?

Reply:
Yeah, I would suggest adding 1 topic right now and once you have a final design ready we can extend this if needed :+1: